webclient/js/elems/raflow/people.js

```diff
@@ -305,9 +276,10 @@ window.loadRAPeopleForm = function () {
                     }
 
                     // Convert integer to bool checkboxes fields
-                    updateRABGInfoFormCheckboxes(peopleData);
+                    updateRATransactantFormCheckboxes(peopleData);
 
                     setPeopleLocalData(TMPTCID, peopleData);
+                    console.log(peopleData);
```

Comment:
use `console.debug` instead of `console.log` so you can remember what lines should be removed later which were added for debugging purpose!

Reply:
Yeah, Good.